src/main/java/spring/com/service/impl/UserMapperServiceImpl.java

```diff
@@ -0,0 +1,24 @@
+package spring.com.service.impl;
+
+import org.springframework.stereotype.Service;
+import spring.com.dto.UserResponseDto;
+import spring.com.model.User;
+
+@Service
+public class UserMapperServiceImpl {
+    public static UserResponseDto dtoBoxing(User user) {
```

Comment:
why is this method static?

Reply:
i thought that these would look good, but i fixed these